### DISCUSSION TITLE
.NET Framework 8

### DISCUSSION THREAD

**Discussion by ofaheem1974** [dotnet/runtime#96017] [General]
Hi,

I hope you are doing well. I would like to know about the possibility of using .NET 8.0 for our web apps. It may offer some advantages from a security standpoint. However, some team members may have reservations about upgrading, particularly given the ongoing security updates that Microsoft has released for .NET Framework 4.8, which we are currently using.

To better understand the potential benefits of .NET Framework 8, I would appreciate any documentation or web links that could help me evaluate this option from a security standpoint.

I understand your time is valuable, and I thank you in advance for considering this matter. I am excited to hear from you soon.

Sincerely,

Osama Faheem

**Comment by huoyaoyuan** [comment]
.NET 5+ was rebranded from .NET Core. It's not named as .NET *Framework* x.

I'm not aware of any difference in security standpoint. Microsoft is releasing monthly security updates for both products.

The support policy of versions can be different: https://dotnet.microsoft.com/en-us/platform/support/policy
For .NET 5+, to be supported with latest security updates, you need to upgrade the framework version regularly every 1 or 2 years.

The differences of the two implementations are more about performance, functionality and deployment model.

**Comment by xingzuhui** [comment]
Microsoft is a Great company for open-source .net.
